package.json

```diff
@@ -43,7 +45,10 @@
         "react-router-dom": "^4.2.2",
         "react-router-redux": "^4.0.8",
         "redux": "^3.7.2",
-        "redux-thunk": "^2.2.0"
+        "redux-thunk": "^2.2.0",
+        "webpack": "^3.10.0",
+        "webpack-dev-middleware": "^1.12.0",
```

Comment:
Rebase and update this dependencies listing

Reply:
@NaiRobley I have fixed them.